pkg/distro/rhel/rhel10/ami.go

```diff
@@ -131,3 +131,24 @@ func mkEC2SapImgTypeX86_64(rd *rhel.Distribution) *rhel.ImageType {
 
 	return it
 }
+
+// Azure Confidential VM
+func mkAzureCVMImgType(rd *rhel.Distribution) *rhel.ImageType {
```

Comment:
Should this go to azure.go?

Reply:
It should, yes.  Thanks for catching it :)